e2e/_suites/ingest-manager/stand-alone.go

```diff
@@ -82,7 +83,7 @@ func (sats *StandAloneTestSuite) aStandaloneAgentIsDeployed() error {
 
 func (sats *StandAloneTestSuite) thereIsNewDataInTheIndexFromAgent() error {
 	maxTimeout := time.Duration(queryRetryTimeout) * time.Minute
-	minimumHitsCount := 100
+	minimumHitsCount := 50
```

Comment:
i agree - I might even lower it more... but this is helpful and still keeps high value

Reply:
It takes 10 seconds to reach 50 hits